### PR TITLE
GhcModBear.py: Add DistributionRequirement

### PR DESCRIPTION
**Pull request by incorrectusername**
Closes https://github.com/coala/coala-bears/issues/1357

### CODE REVIEW

**Comment by sahilarora535** [comment]
ack 377555dec25b26bbc8a316f87f4285d0bcddf2a1

File: bears/haskell/GhcModBear.py [hunk 1]
```diff
@@ -15,7 +17,8 @@ class GhcModBear:
     """
 
     LANGUAGES = {'Haskell'}
-    REQUIREMENTS = {CabalRequirement(package='ghc-mod', version='5.6.0')}
+    REQUIREMENTS = {CabalRequirement(package='ghc-mod', version='5.6.0'),
```
Comment (jayvdb) [inline bears/haskell/GhcModBear.py]: You can now use [AnyOneOfRequirements](https://gitlab.com/coala/package_manager/blob/master/dependency_management/requirements/AnyOneOfRequirements.py) to wrap these two requirements in an OR statement.

**Comment by jayvdb** [comment]
ack b3bd7f2

**Comment by jayvdb** [comment]
@rultor merge

**Comment by meetmangukiya** [comment]
@rultor merge

**Pull request merged**